new-chapter.md

```diff
@@ -0,0 +1,37 @@
+# PyLadies Request to Start Chapter Form
+
+Please fill in the following form to the best of your knowledge, and keep in mind that PyLadies is managed on a volunteer basis and the approval or feedback on the application might take a while. We will contact you as soon as we can.
+
+## Organizer
+
+- `your name`
+- `what are your pronouns (e.g. she/her, they/them)?`
+- `your contact email`
+- `username on PyLadies slack (You can join at https://slackin.pyladies.com/.)`
+- `Who are your co-organizer(s) please include their name and pronouns? We advise having at least 2 organizers a chapter to start.`
+- `What are your co-organizer(s) emails?`
+- `What are your co-organizer(s) PyLadies Slack user names?`
+
+## Background
+
+- `How did you hear about PyLadies?`
+- `Have you attended PyLadies meetups in other cities? If yes where?`
+- `How do you and your organizer(s) use Python? Organizer(s) of all background levels are welcome! Understanding your relationship to Python is helpful for PyLadies to support your chapter as you get started.`
+
+## Chapter Information
+
+- `City of new chapter. PyLadies recommends a city specific focus to prevent name collision. If you want to start a chapter with a different affiliation please specify the location and why you are choosing this.`
+- `Country of new chapter`
+- `Desired namespace for chapter, this will be for the email and subdomain setup like [sf@pyladies.com] ahd it should be as close to the chapter name as possible. Remember, in Python, explicit is better than implicit!`
```

Comment:
```suggestion
- `Desired namespace for chapter, this will be for the email and subdomain setup like [sf@pyladies.com] and it should be as close to the chapter name as possible. Remember, in Python, explicit is better than implicit!`
```